modules/nodes-pods-priority-about.adoc

```diff
@@ -12,7 +12,7 @@ When you use the Pod Priority and Preemption feature, the scheduler orders pendi
 
 You can assign pods a priority class, which is a non-namespaced object that defines a mapping from a name to the integer value of the priority. The higher the value, the higher the priority.
 
-A priority class object can take any 32-bit integer value smaller than or equal to 1000000000 (one billion). Reserve numbers larger than one billion for critical pods that should not be preempted or evicted. By default, {product-title} has two reserved priority classes for critical system pods to have guaranteed scheduling.
+A priority class object can take any 32-bit integer value smaller than or equal to 1000000000 (one billion). Reserve numbers larger than or equal to one billion for critical pods that must not preempt or evict. By default, {product-title} has two reserved priority classes for critical system pods to have guaranteed scheduling.
```

Comment:
’preempt or evict‘ should be changed as 'be preempted or evicted'

Reply:
acked